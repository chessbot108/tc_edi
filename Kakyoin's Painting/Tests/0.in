5
1
2
3
420
696969
