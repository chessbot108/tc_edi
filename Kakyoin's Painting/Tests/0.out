1
13
154
513291815
774841820
